### PR TITLE
[FR] fix links

### PR DESCRIPTION
**Pull request by guillaumebernard84**
### Describe your changes
Fix links before merging to main

### Related issue number or link (ex: `resolves #issue-number`)
needed before merging this PR: https://github.com/cncf/glossary/pull/3270

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by seb-835 — APPROVED** [review]
LGTM 🚀

File: content/fr/chaos-engineering.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ les techniques visant à accroître la résilience et la [fiabilité](/fr/reliab
 La capacité d'un système à tolérer les défaillances tout en assurant une qualité de service adéquate est
 généralement une exigence du développement logiciel.
 Plusieurs aspects sont susceptibles d'entraîner des pannes d'une application,
-comme l'infrastructure, la plateforme ou d'autres parties d'une application ([microservice](/fr/microservices/)).
+comme l'infrastructure, la plateforme ou d'autres parties d'une application ([microservice](/fr/microservices-architecture/).
```
Comment (sestegra) [inline content/fr/chaos-engineering.md:18]: Il manque une parenthèse fermante

**Review by sestegra — CHANGES_REQUESTED** [review]

File: content/fr/chaos-engineering.md (hunk 1 above)
Comment (guillaumebernard84) [reply content/fr/chaos-engineering.md]: bien vu, c'est corrigé

**Review by sestegra — APPROVED** [review]
LGTM 🚀

**Pull request merged**